src/SingleSelectOption.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import propTypes from 'prop-types'
+import cx from 'classnames'
+import { colors, spacers } from './theme.js'
+
+const SingleSelectOption = ({
+    label,
+    active,
+    disabled,
+    onClick,
+    className,
+}) => (
+    <a
+        className={cx(className, {
+            disabled,
+            active,
+        })}
+        onClick={onClick}
+    >
+        {label}
+
+        <style jsx>{`
+            a {
+                display: block;
+                cursor: pointer;
+                font-size: 14px;
+                text-decoration: none;
+                color: ${colors.grey900};
+                padding: ${spacers.dp8} ${spacers.dp12};
+                white-space: nowrap;
+                overflow: hidden;
+                text-overflow: ellipsis;
+            }
+
+            a:hover {
+                background-color: ${colors.grey200};
+            }
+
+            a:active,
+            a.active {
+                background-color: ${colors.grey400};
```

Comment:
Please change to `background-color: ${colors.teal700};` and add `color: ${colors.white};`

Reply:
👍 Changed in 2958c2cf4ef1c8db1a27376f270f4a96f2475c9f